Gemfile

```diff
@@ -25,7 +25,7 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 
 gem 'dotenv-rails', '>= 2.0.1'
 
-gem 'activeadmin', github: 'activeadmin'
+gem 'activeadmin', '1.0.0.pre1'
```

Comment:
Changed this to `pre1` to accommodate a gem I didn't end up actually using, so I might move us back to `pre2` before merging.